examples/producer.py

```diff
@@ -5,7 +5,7 @@
 import json
 import random
 
-print(('pika version: %s') % pika.__version__)
```

Comment:
Could these changes be part of a separate PR? They aren't directly related to dropping `2.6` and `3.3` support, right?